docs/02_cleos/03_command-reference/validate/validate-signatures.md

```diff
@@ -49,3 +49,10 @@ cleos validate signatures transaction '{ "expiration": "2020-04-23T04:47:23", "r
 
 ## See Also
 - [How to submit a transaction](../../02_how-to-guides/how-to-submit-a-transaction.md)
+
+## Output
+```console
```

Comment:
By the way, when do you use `console` here and when do you use `json`. I notice some other examples use `json` when the output is JSON.

Reply:
When it is the output of a command we should use `console` since it could mix JSON with regular output. In this case we could use either `json` or `console`.

Reply:
Probably a good idea to keep `console` here then. Because if no `--chain-id` is provided it will print the following to stderr:
```console
info  2020-05-14T23:27:08.309 thread-0  main.cpp:2647                 operator()           ] grabbing chain_id from nodeos
```